src/app/public/modules/checkbox/checkbox.component.ts

```diff
@@ -126,7 +140,7 @@ export class SkyCheckboxComponent implements ControlValueAccessor, OnInit {
   }
 
   /**
-   * Indicates whether the checkbox is selected.
+   * Indicates whether to the checkbox is selected.
```

Comment:
Did this `to` sneak in here accidentally?

Reply:
good catch, fixed!